src/Bicep.Cli/Commands/DecompileParamsCommand.cs

```diff
@@ -63,7 +63,7 @@ public int Run(DecompileParamsArguments args)
             }
 
             // return non-zero exit code on errors
-            return diagnosticLogger.ErrorCount > 0 ? 1 : 0;
+            return 0;
```

Comment:
Should this command continue to return a non-zero exit code if any errors were raised to match the behavior of `DecompileCommand`?

Reply:
I think that might make sense, but I wanted to avoid changing the existing behavior here. The `diagnosticLogger` isn't invoked at all in this command, so this would always return 0. I've updated the command to clarify this.

Reply:
Makes sense; thanks for removing the misleading comment, too!